src/uu/tr/src/operation.rs

```diff
@@ -339,6 +339,32 @@ impl Sequence {
 
 pub trait SymbolTranslator {
     fn translate(&mut self, current: u8) -> Option<u8>;
+
+    /// Takes two SymbolTranslators and creates a new SymbolTranslator over both in sequence.
+    /// This behaves pretty much identical to `Iterator::chain`:
+    /// <https://doc.rust-lang.org/std/iter/trait.Iterator.html#method.chain>
```

Comment:
```suggestion
    ///
    /// This behaves pretty much identical to [`Iterator::chain`].
```

The newline helps rustdoc with formatting and it can figure out the link with just `[]`.